tests/rules.1.yaml

```diff
@@ -34,3 +34,21 @@ tests:
       periodic: 5
       until: test_traffic.complete
       after: deploy
+- name: test run_action
+  description: Test run_action
+  rules:
+    - do:
+        task: matrix.tasks.deploy
+        entity: cs:bundle/beats-core
+        version: current
+    - do:
+        task: matrix.tasks.health
+      periodic: 5
+      until: run_action.complete
+    - do:
+        task: matrix.tasks.run_action
+        application: kibana
```

Comment:
It does actually deploy it, but it's not blocking properly on the health check.  I'll look in to it.